src/Core/src/Handlers/View/AndroidBatchPropertyMapper.cs

```diff
@@ -1,56 +1,51 @@
-﻿using System;
+﻿#if ANDROID
+
+using System;
 using System.Collections.Generic;
 
-namespace Microsoft.Maui.Handlers
-{
-#if ANDROID
-	class AndroidBatchPropertyMapper<TVirtualView, TViewHandler> : PropertyMapper<TVirtualView, TViewHandler>
-		where TVirtualView : IElement
-		where TViewHandler : IElementHandler
-	{
-		// During mass property updates, this list of properties will be skipped
-		public static HashSet<string> SkipList = new(StringComparer.Ordinal)
```

Comment:
Moved outside because otherwise we have a `SkipList` instance for each generic type used.